docs/parameters.md

```diff
@@ -75,3 +75,42 @@ Boolean parameter evaluation is based on the [values specified in YAML 1.1][http
 * true: `y` `yes` `true` `on`
 * false: `n` `no` `false` `off`
 * also capitalized or uppercase versions of any of these
+
+### steps
+
+Used when you have a job or command that wants to mix predefined and user defined steps. When passed in to a command or job invocation, the steps are always defined as an array, even if only one step is provided.
```

Comment:
Because "steps" is both the command word and the parameter type, in our write-ups we tried to distinguish between the two by calling steps within a parameter "parameter-steps", and the existing list of steps in a command or job just "steps".

I think your descritpion is generally good, though. 👌 example